Extensions/PopUp.json

```
@@ -0,0 +1,282 @@
+{
+  "author": "",
+  "category": "Input",
+  "description": "This extension gives access to three pop-ups. Those are Alert, Comfirm and prompt (text entry).\n## Alert\n- Gives user can alert which they can accept and close.\n- User can respond with `Ok`.\n\n## Confirmation\n- Gives user a confirmation message.\n- User can respond with `Ok` or `Cancel`.\n\n## Prompt\n**Prompt only works in browsers**.\n- Gives user a prompt with a textbox to type in .\n- User can type in the textbox (The typed string is considered the response) and click `Ok` or `Cancel`.\n",
```

Alert prompt and confirm have inconsistent casing. Respect English, write it lowercase unless it is at the start of the sentence.

---

Also, a sentence in parenthesis is not considered a new phrase, so unless it is just after a dot, the first letter in the phrase there is lowercase. There should be no space between the last word of a sentence and the dot. Actually, let me just
```suggestion
  "description": "This extension adds actions to display three kinds of pop-ups.\n\n## Alert\n - Pops-up a message that will pause the game until dismissed by the player.\n\n## Confirmation\n - Pops-up a dialogue containing a message with the option to either confirm or cancel.\n\n## Prompt\n**Note that this pop-up type is unsupported on previews and PC exports.**\n - Displays a message to the player with a text field to write an answer in. The player will also have the option to either confirm or discard their input.\n",
```